src/containers/ImportResources/ImportResources.scss

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+@import '~carbon-components/scss/globals/scss/vars';
+
+.bx--text-input__field-wrapper {
```

Comment:
These `.bx-*` styles at the top level will affect the entire application, is that intended? If so they should be extracted to a _global.scss or similar that's imported from App.scss